<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureWindow</class>
 <widget class="QDialog" name="TemperatureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(243, 243, 243);
background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QLineEdit" name="InputLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
   <item>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Celsius</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kelvin</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="OutputLine">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckBMIButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Check!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
   </property>
   <item>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Celsius</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kelvin</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>TemperatureWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
